common/persistence/persistence_rate_limited_clients.go

```diff
@@ -1052,8 +1078,8 @@ func (p *nexusIncomingServiceRateLimitedPersistenceClient) ListNexusIncomingServ
 	ctx context.Context,
 	request *ListNexusIncomingServicesRequest,
 ) (*ListNexusIncomingServicesResponse, error) {
-	if ok := allow(ctx, "ListNexusIncomingServices", CallerSegmentMissing, p.rateLimiter); !ok {
-		return nil, ErrPersistenceLimitExceeded
+	if err := allow(ctx, "ListNexusIncomingServices", CallerSegmentMissing, p.systemRateLimiter, p.namespaceRateLimiter); err != nil {
```

Comment:
This diff is not applied when merged. lint error on this line does not make any sense!